fastlane/Fastfile

```diff
@@ -180,6 +180,37 @@ platform :ios do
     )
   end
 
+  desc "Change version number and create PR with changes"
+  lane :change_version do |options|
+
+    size = options[:size]
+    case size
```

Comment:
I haven't figured out where but it seems like this size is getting dropped somewhere. All the PRs that open are patch bumps, regardless of what I select in the Jenkins dropdown.

Reply:
This should now be working, after some adjustments.